jax/experimental/jax2tf/examples/tflite/mnist/mnist.py

```diff
@@ -0,0 +1,263 @@
+  batch_metrics_np = jax.device_get(batch_metrics)
+  epoch_metrics_np = {
+      k: np.mean([metrics[k] for metrics in batch_metrics_np])
+      for k in batch_metrics_np[0]}
+
+  print('train epoch: %d, loss: %.4f, accuracy: %.2f', epoch,
+        epoch_metrics_np['loss'], epoch_metrics_np['accuracy'] * 100)
+
+  return optimizer, epoch_metrics_np
+
+
+def eval_model(model, test_ds):
+  metrics = eval_step(model, test_ds)
+  metrics = jax.device_get(metrics)
+  summary = jax.tree_map(lambda x: x.item(), metrics)
+  return summary['loss'], summary['accuracy']
+
+
+def train(train_ds):
+  """Execute model training and evaluation loop.
+
+  Returns:
+    The trained optimizer.
+  """
+  rng = random.PRNGKey(0)
+
+  num_epochs = 3
+  momentum_mass = 0.9
+  learning_rate = 0.1
+  batch_size = 128
+
+  rng, init_rng = random.split(rng)
+  params = get_initial_params(init_rng)
+  optimizer = create_optimizer(params, learning_rate, momentum_mass)
+
+  for epoch in range(1, num_epochs + 1):
+    rng, input_rng = random.split(rng)
+    optimizer, _ = train_epoch(
+        optimizer, train_ds, batch_size, epoch, input_rng)
+  return optimizer
+
+
+# A helper function to evaluate the TF Lite model using "test" dataset.
+def evaluate_tflite_model(tflite_model, test_ds):
+  # Initialize TFLite interpreter using the model.
+  interpreter = tf.lite.Interpreter(model_content=tflite_model)
+  interpreter.allocate_tensors()
+  input_tensor_index = interpreter.get_input_details()[0]['index']
+  output = interpreter.tensor(interpreter.get_output_details()[0]['index'])
+
+  # Run predictions on every image in the "test" dataset.
+  prediction_digits = []
+  for test_image in test_ds['image']:
+    # Pre-processing: add batch dimension and convert to float32 to match with
+    # the model's input data format.
+    test_image = np.expand_dims(test_image, axis=0).astype(np.float32)
+    interpreter.set_tensor(input_tensor_index, test_image)
+
+    # Run inference.
+    interpreter.invoke()
+
+    # Post-processing: remove batch dimension and find the digit with highest
+    # probability.
+    digit = np.argmax(output()[0])
+    prediction_digits.append(digit)
+
+  # Compare prediction results with ground truth labels to calculate accuracy.
+  accurate_count = 0
+  for index in range(len(prediction_digits)):
+    if prediction_digits[index] == test_ds['label'][index]:
+      accurate_count += 1
+  accuracy = accurate_count * 1.0 / len(prediction_digits)
+
+  return accuracy
+
+
+def main(_):
+  train_ds, test_ds = get_datasets()
+  optimizer = train(train_ds)
+  def predict(image):
+    return CNN().apply({'params': optimizer.target}, image)
+
+  # Convert Flax model to TF function.
+  tf_predict = tf.function(
+      jax2tf.convert(predict, with_gradient=False, enable_xla=False),
+      input_signature=[
+          tf.TensorSpec(shape=[1, 28, 28, 1], dtype=tf.float32, name='input')
+      ],
+      autograph=False)
+
+  # Convert TF function to TF Lite format.
```

Comment:
Small suggestion - nitting:

```suggestion
  # Convert the TF function to the TF Lite format.
```

(for consistency)